src/AvtaleSide/NesteForrige/NesteForrige.less

```diff
@@ -3,6 +3,22 @@
     &__nesteknapp {
         margin-left: auto;
     }
+    &__mypadding {
```

Comment:
Ikke et godt navn på klasse, og er den nødvendig?

Reply:
Navn skal endres. 
Det er for å få knapp til god størrelse, var for stor.